src/client/v2/algod/getApplicationBoxByName.ts

```diff
@@ -0,0 +1,23 @@
+import JSONRequest from '../jsonrequest';
+import HTTPClient from '../../client';
+import IntDecoding from '../../../types/intDecoding';
+
+export default class GetApplicationBoxByName extends JSONRequest {
+  constructor(c: HTTPClient, intDecoding: IntDecoding, private index: number) {
+    super(c, intDecoding);
+    this.index = index;
+  }
+
+  path() {
+    return `/v2/applications/${this.index}/box`;
+  }
+
+  // name sets the box name in base64 encoded format.
+  name(name: Uint8Array) {
```

Comment:
Good question - We discussed a bit about it offline last week and came to two thoughts:
1. Since `appArgs` are in `Uint8Array` in this SDK, it makes sense for box name to mimic the JS SDK format (rather than mimic the goal format)
2. It's out of scope for this PR to include these conversion tools for now. We may revisit this in the future and provide functions to convert `int`s or `addr`s later for box names.

Reply:
Makes sense. Thanks for the explanation!